recipes/igdiscover/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - snakemake >=3.9.0
     - cutadapt
     - xopen >=0.1.1
-    - seaborn >=0.6.0
+    - seaborn =0.7
```

Comment:
Any particular reason for that? I don’t really care as long as I get 0.7.1 most of the time, which is the most recent working version.